conf/prod.routes

```diff
@@ -12,6 +12,14 @@ GET        /admin/metrics                             @com.kenshoo.play.metrics.
 
 POST       /email-verification/verification-requests  @uk.gov.hmrc.helptosavestub.controllers.EmailVerificationController.verify
 
-GET       /v1/isEligible/:nino                        @uk.gov.hmrc.helptosavestub.controllers.DWPController.dwpClaimantCheck(nino: String, systemId: String, thresholdAmount: Int, transactionId: Option[java.util.UUID])
+GET        /v1/isEligible/:nino                        @uk.gov.hmrc.helptosavestub.controllers.DWPController.dwpClaimantCheck(nino: String, systemId: String, thresholdAmount: Int, transactionId: Option[java.util.UUID])
 
 GET        /pay-as-you-earn/02.00.00/individuals/:nino        @uk.gov.hmrc.helptosavestub.controllers.PayePersonalDetailsController.getPayeDetails(nino)
+
+GET        /help-to-save-proxy/nsi-services/account                      uk.gov.hmrc.helptosavestub.controllers.NSIController.getAccount(correlationId: Option[String])
```

Comment:
Assuming this is a stub of NS&I, not a stub of help-to-save-proxy, these URLs shouldn't have the `/help-to-save-proxy/` prefix

Reply:
I understand now, thanks